src/select.dust

```diff
@@ -0,0 +1,3 @@
+<div class="c-select">
+    {select}
+</div>
```

Comment:
Don’t in any way consider this a blocker, but at some point we should probably be using the dust template in the tests.